interfaces/apparmor/template.go

```diff
@@ -478,3 +478,13 @@ var classicJailmodeSnippet = `
   # This is related to LP: #1666897
   @{INSTALL_DIR}/core/*/{,usr/}lib/@{multiarch}/{,**/}lib*.so* m,
 `
+
+// nfsSnippet contains extra permissions necessary for snaps and snap-confine
+// to operate when NFS is used. This is an imperfect solution as this grants
+// some network access to all the snaps on the system.
```

Comment:
I'm honestly quite surprised that the _application_ needs network access to access the filesytem. I won't question that as we'd not have gotten that far if that wasn't true, but I'd like to hear more about how this ended up like this. It seems to make no sense to me at least.

Reply:
I'll find the relevant apparmor bug and link it here.

Reply:
I also added the reference to the bug report into the source code.